doc/whats_new/v1.4.rst

```diff
@@ -246,6 +246,10 @@ Changelog
   :class:`metric.DistanceMetric` objects.
   :pr:`26267` by :user:`Meekail Zain <micky774>`
 
+- |Feature| :func:`neighbors.radius_neighbors_graph` now uses the 
+  `validate_params` decorator for parameter validation. 
+  :pr:`27245` by :user:`Sayed Qaiser Ali <sqali>`
+
```

Comment:
We don't mention the param validation PRs in the changelog because they have no real consequences for the users.
```suggestion
```